contracts/land/LANDRegistry.sol

```diff
@@ -479,4 +472,87 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     assembly { size := extcodesize(addr) }
     return size > 0;
   }
+
+  //
+  // LAND Ping
+  //
+
+  /**
+   * @dev Set date from when the LAND Ping feature should be enabled
+   * @param _gracePeriod Desired amount of time in miliseconds to enable feature
+   */
+  function setGracePeriod(uint _gracePeriod) external onlyDeployer {
+    gracePeriod = _gracePeriod;
```

Comment:
I'm thinking if we should add a `require` here to check that the grace period has been set to ensure that won't be updated.

Anyways the contract is upgradable.

What do you think @eordano @abarmat?